lib/src/ui/game/archived_game_screen.dart

```diff
@@ -76,6 +81,41 @@ class ArchivedGameScreen extends ConsumerWidget {
   }
 }
 
+class _AppTitle extends ConsumerWidget {
```

Comment:
This shouldn't be there, it's not part of the material difference feature, so please remove it.

I'd like to review features/improvements separately, and I'm not sure to accept this change.